micrometer-java11/build.gradle

```diff
@@ -0,0 +1,24 @@
+description 'Micrometer core classes that require Java 11'
```

Comment:
I made the description more generic. We could add other things to this module in the future. One example would be an implementation of our HttpClient abstraction based on the JDK 11 HttpClient.